examples/README.md

```diff
@@ -0,0 +1,46 @@
+# Overview
+
+This file contains specific usage instructions and additional information used for running a particular `example` app.
+
+## Examples
+
+### USB-I2C-MCTP
+
+This application requires that the kernel console buffer sizes are increased as well as i2c-master-slave driver buffers. This is because we are doing transfers of sizes greater than
+what the upstream kernel is allowed to do.
+
+The following reference can be used to prepare the kernel for the uart/console buffers. The size specified here must be >= RX_BUF_LEN (as described by the app).
+
+```
+diff --git a/capsules/core/src/virtualizers/virtual_uart.rs b/capsules/core/src/virtualizers/virtual_uart.rs
+index 0d39fe024..59e0435a1 100644
+--- a/capsules/core/src/virtualizers/virtual_uart.rs
++++ b/capsules/core/src/virtualizers/virtual_uart.rs
+@@ -54,7 +54,7 @@ use kernel::hil::uart;
+ use kernel::utilities::cells::{OptionalCell, TakeCell};
+ use kernel::ErrorCode;
+ 
+-pub const RX_BUF_LEN: usize = 64;
++pub const RX_BUF_LEN: usize = 132;
+ 
+ pub struct MuxUart<'a> {
+     uart: &'a dyn uart::Uart<'a>,
+```
```

Comment:
I think just put this in the `//!` comments for the example. Otherwise looks good